src/webview_provider/newLineagePanel.ts

```diff
@@ -814,6 +848,13 @@ export class NewLineagePanel implements LineagePanelView {
     };
   }
 
+  private isExternalProject(
+    node: NodeMetaData | SourceMetaData,
+    projectName: string,
+  ) {
+    return node.package_name !== projectName;
```

Comment:
Only package_name has information about the project. I am assuming we dont show nodes from installed dependency packages. @mdesmet pls check if this ok

Reply:
We do show nodes from installed packages. This is not correct.

Reply:
Also this business logic should be returned by the parser instead of doing it in a consumer of the metadata